ui/helpers/constants/gas.js

```diff
@@ -31,3 +31,11 @@ export function getGasFormErrorText(type, t, { minimumGasLimit } = {}) {
       return '';
   }
 }
+
+export const GasLevelIconMap = {
```

Comment:
nit: Since this isn't a class, and in keeping with naming, should this be `priorityLevelIconMap` or `PRIORITY_LEVEL_ICON_MAP`?

Reply:
Good point, I changed it to `PRIORITY_LEVEL_ICON_MAP`